drv/stm32xx-sys/src/main.rs

```diff
@@ -690,17 +715,16 @@ impl idl::InOrderSysImpl for ServerImpl<'_> {
 
                             if mask & enable_mask != 0 {
                                 // Record that these bits meant something.
-                                used_bits |= mask;
+                                let bit = 1 << i;
```

Comment:
Ought to be able to factor `bit` here out (it's duplicated on line 701)

Reply:
I wanted to avoid doing the bit shift in the case where we don't do anything with the entry. But, bit shifts are cheap...right?